.github/dependabot.yml

```
@@ -0,0 +1,20 @@
+version: 2
+
+updates:
+  - package-ecosystem: "composer"
```

```suggestion
  - package-ecosystem: "composer1"
```
I think this will force composer v1 which is what we need right now if it defaults to v2

---

Looking at their docs it was not very clear how to do this. Found this https://github.com/dependabot/dependabot-core/pull/2838#issuecomment-744436945 and it notes it should auto-detect based on a composer.lock composer-plugin-api setting, which look like we can now set as part of composer.json.

So instead, I _think_ we should add this to our composer.json.

```"require": {
        "composer-plugin-api": "^1.10"
    },
    "require-dev": {
        "composer/composer": "^1.10"
    },```

Thoughts?

---

I'm game with that if it works